lib/ui/view/channel/edit_channel/editChannelViewModel.dart

```diff
@@ -1,6 +1,128 @@
+import 'package:flutter/cupertino.dart';
+import 'package:hng/app/app.locator.dart';
```

Comment:
Use of material  or Cupertinowidgets isn't allowed in view model.. check form view for any text editing controller in stacked architecture